plugins/woocommerce/changelog/56431-add-product-details-available-in-product-context-only

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Extract query loop product context validation into reusable HOC.
```

Comment:
```suggestion
Extract query loop product context validation into a hook.
```

Reply:
@cpapazoglou Thanks for pointing this out. Actually I plan to update the PR description after receiving feedback from @louwie17, which will automatically update this change file. 🤝